src/platform/packages/shared/kbn-workflows/spec/lib/generate_yaml_schema_from_connectors.ts

```diff
@@ -22,6 +22,18 @@ import {
   WorkflowSchema,
 } from '../schema';
 
+/**
+ * Backward compatibility: Map old type names to new cleaner type names.
+ * These aliases are accepted in schema validation but not shown in autocomplete.
+ * The key is the old type name, the value is the new type name.
+ */
+const KIBANA_TYPE_ALIASES: Record<string, string> = {
```

Comment:
we have basically the same info in `KIBANA_TYPE_ALIASES`, in `OPERATION_TYPE_OVERRIDES ` and in `KIBANA_TYPE_ALIASES `. can we instead declare it in spec/kibana/aliases.ts as a single source of truth and use everywhere?